Avoid astropy model fitting error if all uncertainties are 0 (#1880)
spacetelescope/jdaviz: CHANGES.rst

Bug Fixes
---------

- Avoid a non-finite error in model fitting by not passing spectrum uncertainties as

>   weights if there are uncertainty values of 0. [#1880]

Reviewer: ```suggestion
  weights if the uncertainty values are all 0. [#1880]
```
